src/NuGet.Core/NuGet.PackageManagement/IDE/IPackageRestoreManager.cs

```diff
@@ -50,8 +50,10 @@ public interface IPackageRestoreManager
         /// <param name="solutionDirectory">Current solution directory</param>
         /// <param name="packageReferencesDict">Dictionary of package reference with project names</param>
         /// <returns>List of packages restore data with missing package details.</returns>
-	    IEnumerable<PackageRestoreData> GetPackagesRestoreData(string solutionDirectory,
+#pragma warning disable IDE0055
```

Comment:
I don't why IDE0055 (Fix formatting) analyzer reported a warning for the following method in the interface. Adding a suppression to GlobalSuppressions.cs file didn't resolve the warning. Hence added the suppression directly in the code.